specs/lib/MergeDuplicatePropertiesSpec.js

```diff
@@ -139,8 +139,6 @@ describe('MergeDuplicateProperties', function() {
 
     var mergeShaders = MergeDuplicateProperties.mergeShaders;
     describe('mergeShaders', function() {
-        var testShaderBufferOne = Buffer.from('test shader one', 'utf8');
-        var testShaderBufferTwo = Buffer.from('test shader two', 'utf8');
         it('merges duplicate shaders', function() {
```

Comment:
I agree here. In general we have strayed away from creating too many test gltf files on disk, but maybe a middle ground is good.